### PR TITLE
chore(textbox): remove classic theme - FE-3355 

### PR DESCRIPTION
**Pull request by mkrds**
BREAKING CHANGE: Textbox no longer supports classic theme

This PR removes classic theme from `Textbox` and refactors stories to use new `CSF` / `MDX` formats

### Checklist

- [x] Commits follow our style guide
- [x] Screenshots are included in the PR if useful
- [x] All themes are supported if required
- [x] Unit tests added or updated if required
- [x] Cypress automation tests added or updated if required
- [x] Storybook added or updated if required
- [x] Typescript `d.ts` file added or updated if required
- [x] Carbon implementation and Design System documentation are congruent

### CODE REVIEW

**Review by sjohnson-sage — APPROVED** [review]

**Review by edleeks87 — DISMISSED** [review]

File: cypress/features/regression/experimental/textboxMultiple.feature [hunk 1]
```diff
@@ -12,27 +12,27 @@ Feature: Experimental Textbox multiple component
 
   @positive
   Scenario: Check disabled checkbox for a Textbox multiple component
-    When I open multiple "Experimental-Textbox" component in noIFrame with "textbox" json from "experimental" using "disabled" object name
+    When I open multiple "Experimental Textbox Test" component in noIFrame with "textbox" json from "experimental" using "disabled" object name
```
Comment (KatarzynaQA) [inline cypress/features/regression/experimental/textboxMultiple.feature:15]: All tests as `disabled`, `readonly` etc which are covered by chromatic should be delete

**Review by KatarzynaQA — DISMISSED** [review]

**Review by KatarzynaQA — DISMISSED** [review]

**Review by sjohnson-sage — DISMISSED** [review]

**Review by edleeks87 — APPROVED** [review]

**Comment by Alec-Walker-Sage** [comment]
Props not appearing in the MDX example on storybook

**Comment by mkrds** [comment]
@Alec-Walker-Sage Fixed

**Review by Alec-Walker-Sage — APPROVED** [review]

**Review by edleeks87 — APPROVED** [review]

**Pull request merged**